### DISCUSSION TITLE
Basis Trading

### DISCUSSION THREAD

**Discussion by alenpavlovich** [kernc/backtesting.py#430] [Q&A]
hi team,

If I wanna trade futures premium (buy spot, sell futures), is it possible to track position on both instruments with one strategy?

**Comment by kernc** [comment]
You could try the approach described in https://github.com/kernc/backtesting.py/issues/104#issuecomment-658779951.

**Reply by alenpavlovich** [reply]
But how can I keep track of my positions? I need to keep track of two positions in this case.
